README_CN.md

```diff
@@ -0,0 +1,621 @@
+
+10. 如果使用 MacOS/Linux 操作系统，请选择系统 Python 可执行文件来在 ESP-IDF 工具内创建 ESP-IDF 虚拟环境，并安装 ESP-IDF Python 包。
+
+    > **注意：** Windows 用户不需要选择 Python 可执行文件，因为此设置过程会自动安装所需文件。
+
+11. 确保 `IDF_TOOLS_PATH` 中不包含空格，避免构建过程中出现问题，且 `IDF_TOOLS_PATH` 与 `IDF_PATH` 不能相同。
+
+12. 此时应出现安装界面，显示设置进度状态，包括 ESP-IDF 下载进度、ESP-IDF 工具的下载和安装进度，以及 Python 虚拟环境的创建过程。
+
+13. 如果一切正常，将收到所有设置已配置完成的消息，此时可开始使用扩展。
+
+如有问题，请参阅[故障排除](#Troubleshooting)部分。
+
+##  在 VSCode 中使用 ESP-IDF 扩展进行开发
+
+ESP-IDF 扩展在 VS Code 底部蓝色窗口的状态栏中提供了一系列命令图标，将鼠标悬停在图标上时，会看到可执行的命令。
+
+<p>
+  <img src="./media/readme/statusBar.png" alt="状态栏">
+</p>
+
+以下步骤展示了这些图标的常见用例：
+
+1. 按 <kbd>F1</kbd> 并输入 **ESP-IDF: Show Examples Projects**，可以基于 ESP-IDF 示例创建新项目。在命令面板中选择 ESP-IDF，搜索想使用的示例并创建新项目。
+
+2. 创建好新项目并在 VS Code 中打开后，点击状态栏图标 ![串行端口](./media/readme/serialport.png) 设置设备的串行端口。也可以按 <kbd>F1</kbd> 输入 **ESP-IDF: Select Port to Use:**，选择设备连接的串行端口。
+
+3. 点击状态栏图标 ![IDF 目标](./media/readme/target.png) 选择使用的芯片设备（如 esp32、esp32s2 等），或按 <kbd>F1</kbd> 输入 **ESP-IDF: Set Espressif Device Target** 命令。
+
+4. 接下来，通过点击状态栏图标 ![sdkconfig 编辑器](./media/readme/sdkconfig.png) 或按 <kbd>F1</kbd> 输入 **ESP-IDF: SDK Configuration Editor** 命令（快捷键：<kbd>CTRL</kbd> <kbd>E</kbd> <kbd>G</kbd>），修改 ESP-IDF 项目设置。完成所有更改后，点击 `Save` 并关闭此窗口。可以选择菜单栏中的`查看` -> `输出`，并从下拉列表中选择 `ESP-IDF`。
+
+5. （可选）运行 **ESP-IDF: Run idf.py reconfigure task** 命令生成 `compile_commands.json` 文件，以便启用语言支持。也可以按照 [C/C++ 配置](./docs/C_CPP_CONFIGURATION.md) 文档中的说明配置 `.vscode/c_cpp_properties.json`。
+
+6. 请自行对代码进行必要修改。完成项目后，点击状态栏图标 ![构建](./media/readme/build.png) 或按 <kbd>F1</kbd> 输入 **ESP-IDF: Build your Project** 来构建项目。
+
+7. 点击状态栏图标 ![烧录](./media/readme/flash.png) 或按 <kbd>F1</kbd> 输入 **ESP-IDF: Flash your project**，依据使用的接口类型，在命令面板中选择 `UART`、`DFU` 或 `JTAG`，将应用程序烧录到设备上。
+
+8. 点击状态栏图标 ![烧录方式](./media/readme/flashmethod.png) 或按 <kbd>F1</kbd> 输入 **ESP-IDF: Select Flash Method**，从 `UART`、`DFU` 或 `JTAG` 中选择想要更改的烧录方式。也可以直接使用命令 **ESP-IDF: Flash (UART) your Project**、**ESP-IDF: Flash (with JTag)** 或 **ESP-IDF: Flash (DFU) your project**。
+
+9. 点击状态栏图标 ![监视](./media/readme/monitor.png) 或按 <kbd>F1</kbd> 输入 **ESP-IDF: Monitor Device** 启动监视器，在 VS Code 终端中记录设备活动。
+
+10. 根据 ESP-IDF 文档中的要求来配置驱动程序，详情请参考[配置 JTAG 接口](https://docs.espressif.com/projects/esp-idf/zh_CN/latest/esp32/api-guides/jtag-debugging/configure-ft2232h-jtag.html)。
+
+11. 在调试设备之前，先按 <kbd>F1</kbd> 输入 **ESP-IDF: Select OpenOCD Board Configuration**，选择设备的 OpenOCD 开发板配置文件。点击状态栏图标 ![openocd](./media/readme/openocd.png) 或按 <kbd>F1</kbd> 输入 **ESP-IDF: OpenOCD Manager** 命令来测试连接。可以选择菜单栏中的`查看` -> `输出`，并从下拉列表中选择 `ESP-IDF`。
+
+    > **注意：** 可以使用 **ESP-IDF: OpenOCD Manager** 命令或者点击 VS Code 状态栏中的 `OpenOCD Server (Running | Stopped)` 按钮来启动或停止 OpenOCD。
+
+12. 确保项目已经构建并烧录，OpenOCD 连接正常，调试器能正常工作。按 <kbd>F5</kbd> 启动调试会话。调试会话的输出可在菜单栏中选择`查看` -> `调试控制台`进行查看。
+
+如有问题，请参阅[故障排除](#Troubleshooting)部分。
+
+# 拓展阅读
+
+您可以在下面找到关于所有功能的教程、命令和文档的详细列表。
+
+- 查看有关 ESP-IDF 扩展的[乐鑫文档](https://docs.espressif.com/projects/vscode-esp-idf-extension/en/latest/).
```

Comment:
For now, this page only have English version. If we replace "en" in the link with "zh_CN", it will turn 404. But in case we forget to edit this link when CN version is available, I will  accept your suggestion. Thx!

Reply:
Hi @Shen765 sorry for the confusion caused. I mean, we should use "。" instead of "." for this line and the lines in the same bullet list below it~
```suggestion
- 查看有关 ESP-IDF 扩展的[乐鑫文档](https://docs.espressif.com/projects/vscode-esp-idf-extension/en/latest/)。
```

Reply:
Hi @April-Yjj Thanks for explanation. I missed the "commas" in your comment and misunderstood your point. Sorry  :(